lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -373,6 +373,20 @@ testRule({
 			description: 'explicit scroll-padding test',
 			message: messages.expected('scroll-padding'),
 		},
+		{
+			code: 'a { grid-template-columns: repeat(3, 1fr); grid-template-rows: auto; grid-template-areas: "left center right"; }',
+			unfixable: true,
+			description:
+				'the repeat() notation has non-trivial semantics and is currently not fixable (columns)',
+			message: messages.expected('grid-template'),
+		},
+		{
+			code: 'a { grid-template-columns: auto; grid-template-rows: repeat(3, 1fr); grid-template-areas: "left center right"; }',
+			unfixable: true,
+			description:
+				'the repeat() notation has non-trivial semantics and is currently not fixable (rows)',
+			message: messages.expected('grid-template'),
+		},
```

Comment:
minor refactor

maybe 1. add the fixed code 2. remove unfixable 3. add `skip: true,`
so if someone ever do fix it, the test case will be ready

Reply:
I think keeping this as-is is better because `skip: true` is ambiguous about whether to mark TODO or skip the case for some reason (e.g. flakiness). If we could modify this unsupported autofix in the future, probably some issue would be opened.